test/system/500-networking.bats

```diff
@@ -957,14 +979,15 @@ function wait_for_restart_count() {
 }
 
 # Test for https://github.com/containers/podman/issues/18615
+# CANNOT BE PARALLELIZED due to strict checking of /run/netns
```

Comment:
I wonder how much value this check here does. I mean I want to check that we do no leak the netns files however we only do so here in a ugly way IMO. Podman uses a consistent netns file name, as such I wonder if we should just add such check in the leak_check  at the end of the suite. Then it will cover all tests not just this one. There should never be reason for us to leak the file, however it would be hard to know where it was leakded in case of failures but I guess if leaks happen we can instrument teardown to ls /run/netns to notice where it starts leaking...

Of course out of scope of this PR.

Reply:
Well.... this comment does have your name on it https://github.com/containers/podman/blob/ae4eaaf1ae6921eab9d5ba5f3c9252cb267e77b1/test/system/500-networking.bats#L1002-L1003

I'm all for adding a leak check, but what would that look like? Right now on my laptop I see six `/run/netns/netns*` files with no tests running. Looks like I have some orphan pasta processes

Reply:
I know this was added by me, my comment was more of if it is blocking parallel tests it may not be a good place because checking leaks in one place doesn't help much if we still leak in other places 

> I'm all for adding a leak check, but what would that look like? Right now on my laptop I see six /run/netns/netns* files with no tests running. Looks like I have some orphan pasta processes

pasta should not be used as root so I don't think they relate to each other. If there are leaks right now then we need to debug/fix them first of course. Anyhow as said not in this PR. I will file a new issue to consider this.